alembic/versions/eb8269f6bb85_bug_1369498_remove_adi.py

```diff
@@ -0,0 +1,38 @@
+"""bug 1369498 remove adi
+
+Remove ADI-related tables and stored procedures.
+
+Revision ID: eb8269f6bb85
+Revises: 0db05da17ae8
```

Comment:
What do `eb8269f6bb85` and `0db05da17ae8` refer to? Are they randomly generated?

Reply:
Ah, cool, thanks.